homeassistant/components/nintendo_parental_controls/sensor.py

```diff
@@ -54,6 +59,23 @@ class NintendoParentalControlsSensorEntityDescription(SensorEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         value_fn=lambda device: device.today_time_remaining,
     ),
+    NintendoParentalControlsSensorEntityDescription(
+        key=NintendoParentalControlsSensor.TIME_EXTENDED,
+        translation_key=NintendoParentalControlsSensor.TIME_EXTENDED,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda device: device.extra_playing_time,
+        available_fn=lambda device: device.extra_playing_time is not None,
+    ),
+    NintendoParentalControlsSensorEntityDescription(
+        key=NintendoParentalControlsSensor.LAST_SYNC,
+        translation_key=NintendoParentalControlsSensor.LAST_SYNC,
+        device_class=SensorDeviceClass.TIMESTAMP,
+        value_fn=lambda device: dt_util.utc_from_timestamp(device.last_sync)
+        if device.last_sync
+        else None,
+    ),
```

Comment:
What does this mean? In general HA tracks the last time data was updated

Reply:
Yeah that's fine 👍